as-controller-board-status/functions/notify_test.go

```diff
@@ -20,14 +25,20 @@ type testTableBuildSubscriptionMessageStruct struct {
 
 func prepBuildSubscriptionMessage() []testTableBuildSubscriptionMessageStruct {
 	commonSuccessConfig := GetCommonSuccessConfig()
+
+	mockAppService := &mocks.ApplicationService{}
+	subscriptionClient := &client_mocks.SubscriptionClient{}
```

Comment:
convention is to name the variable with mock in the name
```suggestion
	mockSubscriptionClient := &client_mocks.SubscriptionClient{}
```